src/gobmanagement/socket.py

```diff
@@ -64,12 +65,14 @@ def _broadcasts(self):
         print("Start broadcast new logs", self._clients)
 
         while self._clients > 0:
-            last_logid = get_last_logid()
+            with session_scope(True) as session:
+                last_logid = get_last_logid(session)
+                last_timestamp = get_last_service_timestamp(session)
```

Comment:
Not that this is changed in this PR, but do you have any idea making this query faster?
It queries for max logid but is count(*) not faster, or is there any other way to improve performance?

Reply:
There is no index on this column I think we should just add one. Counting will always be slower unless we have a tally or use pg_stats (see links below).

Recommend index:
```CREATE INDEX ix_service_timestamp ON services (timestamp DESC)```

Would be interesting to see what the affects are of using a WHERE constraint on the index (in this case, a constraint that only takes timestamps of the last day or hour), assuming that any client subscribed on this information will never be interested in logs older than this constraint.

Relevant:
 * [Last modification date retrieval (stackexchange)](https://dba.stackexchange.com/questions/58214/getting-last-modification-date-of-a-postgresql-database-table)

Also note, same as for logs, we could also just make materialized views that partition the available data.